assembl/static2/js/app/components/debate/navigation/timelineSegment.jsx

```diff
@@ -58,7 +59,15 @@ class TimelineSegment extends React.Component {
     }
   }
   render() {
-    const { index, barWidth, identifier, isCurrentPhase, isStepCompleted, phaseIdentifier, title, locale } = this.props;
+    const { index, barWidth, isCurrentPhase, isStepCompleted, title, locale } = this.props;
+    const timelineClass = () => {
```

Comment:
use `import classNames from 'classnames'`. See an example in `components/debate/common/ideasLevel`